sys/net/gnrc/link_layer/lwmac/README.md

```diff
@@ -0,0 +1,26 @@
+LwMAC implementation
```

Comment:
Could you move this to the detailed description of the `net_gnrc_lwmac` group in Doxygen, please? That would be just below [this line](https://github.com/RIOT-OS/RIOT/pull/6554/files#diff-a69ff1fb011163871bf1b36b5de23f35R14)